angr/state_plugins/sim_action.py

```diff
@@ -141,6 +149,10 @@ def __init__(self, state, constraint, condition=None):
     def all_objects(self):
         return [ a for a in ( self.constraint, self.condition ) if a is not None ]
 
+    @property
+    def is_symbolic(self):
+        return getattr(self.constraint, "symbolic", False)
```

Comment:
Same as above. `getattr()` is such a weird way of accessing a static property in Python.